gocwebtemplate-core/gocwebtemplate-core-base/src/main/java/goc/webtemplate/component/AbstractCoreBean.java

```diff
@@ -2020,15 +2110,29 @@ private AppTop buildAppTop() {
     private IPreFooter buildPreFooter(boolean isTransactional, boolean isUnilingualError) {
         if (!isTransactional) {
             if (!isUnilingualError) {
-                return new PreFooter(

```

Comment:
As discussed:
- FeedbackLink class moved to "frontend" and existing setFeedback methods because depracated and for backwards compatibility simply sets FeedbackLink object properties.  FeedbackLink instance variable is created from config properties similarily to LeavingSecureSite
- Feedback class stays "backend" class and have the same rules for legacyUrl as in .NET
- Put FeedbackLink object creation in its own `buildFeedbackLink` method and call it when building PreFooter object
-

Reply:
done!